ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/operations/Attestation.java

```diff
@@ -53,6 +53,13 @@ public Attestation(
     this.signature = signature;
   }
 
+  public Attestation(Attestation attestation) {
```

Comment:
(optional) A static factory method might be clearer:
```suggestion
  public static Attestation deepCopy(Attestation attestation) {
```

Reply:
I'm gonna pass this one for now because other than BeaconState, we have a similar naming in almost all other data structures, where deep copy is just another constructor